lib/state/simperium/middleware.ts

```diff
@@ -246,27 +246,45 @@ export const initSimperium = (
       case 'FILTER_NOTES':
       case 'OPEN_NOTE':
       case 'SELECT_NOTE': {
+        const state = getState();
```

Comment:
this is already available from above as `prevState` and `nextState` depending on if you want the state before or after the reducers updated. in this case it shouldn't matter because these actions won't affect revisions. might as well use `nextState`